notebooks/403-action-recognition-webcam/README.md

```diff
@@ -0,0 +1,39 @@
+# Live Human Action Recognition with OpenVINO
```

Comment:
```suggestion
# Live Action Recognition with OpenVINO
```
This is my personal suggestion, don't feel like you need to accept it. "Live Human" sounds strange when I said it out loud. I suggest not putting the two words together. Open to alternatives.